app/src/main/java/com/example/ithaca_transit_android_v2/Models/Trip.kt

```diff
@@ -1,5 +1,7 @@
 package com.example.ithaca_transit_android_v2.Models
 
+import com.example.ithaca_transit_android_v2.models.Location
+
```

Comment:
Here, one of the models folder is capitalized and the other is not. I'm not sure if this is an issue that we will need to fix.